webpack.config.js

```diff
@@ -15,10 +17,38 @@ module.exports = {
           options: babelConfig,
         },
       },
+      {
+        test: /\.scss$/i,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              esModule: true,
+              modules: {
+                namedExport: true,
+              },
+            },
+          },
+          'postcss-loader',
+          'sass-loader',
+        ],
+        include: /\.module\.scss$/,
+      },
+      {
+        test: /\.scss$/i,
+        use: [
+          MiniCssExtractPlugin.loader,
+          'css-loader',
+          'postcss-loader',
+          'sass-loader',
+        ],
+        exclude: /\.module\.scss$/,
+      },
```

Comment:
this rule sets up support for `.scss` (excluding SCSS modules, which the above rule is for). i think we'll want/need global SCSS file support for things like fonts, but i'm not sure yet -- should i remove this until we definitely need it?

Reply:
I feel like it makes sense to have - I'm guessing things like color constants would also be in global SCSS files?